src/Header/SetCookie.php

```diff
@@ -618,13 +618,18 @@ public function getSameSite()
      */
     public function setSameSite($sameSite)
     {
-        if ($sameSite !== null && ! in_array($sameSite, self::SAME_SITE_ALLOWED_VALUES, true)) {
-            throw new Exception\InvalidArgumentException(sprintf(
-                'Invalid value provided for SameSite directive: "%s"; expected one of: Strict, Lax or None',
-                is_scalar($sameSite) ? $sameSite : gettype($sameSite)
-            ));
+        if ($sameSite !== null) {
```

Comment:
Early return trick can be used to increase readability by decreasing indentation level and avoiding else statements:
```php
if ($sameSite === null) {
    $this->sameSite = null;
    return;
}
```